lib/controller.php

```diff
@@ -202,7 +215,9 @@ public function export_all() {
 			return;
 		}
 
-		$posts = $wpdb->get_col( "SELECT ID FROM $wpdb->posts WHERE post_status = 'publish' AND post_type IN ('post', 'page' )" );
+		$post = new WordPress_GitHub_Sync_Post(0);
+		$blacklisted_post_types = $post->get_blacklisted_values();
```

Comment:
Instead of instantiating a blank object, make the `get_blacklisted_values` a static method.